docs/site/HasOne-relation.md

```diff
@@ -432,21 +491,21 @@ to show the idea:
 A `hasOne` relation has an `inclusionResolver` function as a property. It
 fetches target models for the given list of source model instances.
 
-Using the relation between `Supplier` and `Account` we have shown above, a
-`Supplier` has one `Account`.
+Use the relation between `Customer` and `Address` we show above, a `Customer`
```

Comment:
I am using `Customer` and `Address` example here because of the [relation-inclusion.png](https://github.com/strongloop/loopback-next/blob/master/docs/site/imgs/relation-inclusion.png) image. And I used `Supplier` and `Account` example above because of [hasOne-relation-example.png](https://github.com/strongloop/loopback-next/blob/master/docs/site/imgs/hasOne-relation-example.png) and [hasOne-relation-graph.png](https://github.com/strongloop/loopback-next/blob/master/docs/site/imgs/hasOne-relation-graph.png). Do I replace  everything with `Customer` and `Address` example?

Reply:
I think `Supplier` and `Account` would be better since the `relation-inclusion.png` is only used for explaining the how the inclusion resolvers are related to the repositories.

Reply:
Alright, thanks